server/engine/engine.py

```diff
@@ -1187,20 +1190,25 @@ def processLocalSubjectFolder(path: str, outputName: str = None, href: str = '')
             if fitDynamics and dynamicsInit is not None:
                 residualForce = trialProcessingResult['linearResidual']
                 residualTorque = trialProcessingResult['angularResidual']
-                f.write(f'  - Avg. Residual Force   = {residualForce:1.2f} N\n')
-                f.write(f'  - Avg. Residual Torque  = {residualTorque:1.2f} N-m\n')
+                f.write(
+                    f'  - Avg. Residual Force   = {residualForce:1.2f} N\n')
+                f.write(
+                    f'  - Avg. Residual Torque  = {residualTorque:1.2f} N-m\n')
```

Comment:
I normally don't care about the auto-formatting too much, but a lot of these changes are adding unnecessary lines. If it's easy to revert, great. If not, don't worry about it.